core/src/main/java/org/apache/accumulo/core/conf/Property.java

```diff
@@ -430,12 +433,33 @@
       "When a tablet server's SimpleTimer thread triggers to check idle"
           + " sessions, this configurable option will be used to evaluate update"
           + " sessions to determine if they can be closed due to inactivity"),
+  @Deprecated
   TSERV_READ_AHEAD_MAXCONCURRENT("tserver.readahead.concurrent.max", "16", PropertyType.COUNT,
-      "The maximum number of concurrent read ahead that will execute. This effectively"
-          + " limits the number of long running scans that can run concurrently per tserver."),
+      "This property is deprecated, use tserver.scan.executors.default.threads instead."),
```

Comment:
Should not remove the old description, because it's still useful to know what it does, if anything. Also, please `s/deprecated/deprecated since 2.0.0/`.